rock/stream_aligner_test_helpers.rb

```diff
@@ -0,0 +1,96 @@
+# frozen-string-literal: true
+
+# no-doc
+module Syskit
+    module Aggregator
+        # Test helpers for stream aligner
+        #
+        # These create a synchronization by using the `transformer_stream_aligner_port`
+        # for transformer tasks or `stream_aligner_status_port` for pure stream aligner
+        # tasks by checking the time of the processed samples until one matches the
+        # written sample time or the timeout is reached.
+        module TestHelpers
+            # @api public
+            #
+            # Writes a sample to the port and wait until a status reader matching
+            # `latest_time` and `sample_time_field` is found.
+            # In addition, let's the caller call more predicates for the function
+            # expect execution.
+            #
+            # @param [task]
+            # @param [port] The port where the sample must be written
+            # @param [sample] The sample that must be written
+            # @param [sample_time_field] Parameter name as a string for
+            #                            the time on the sample
```

Comment:
The syntax is `@param [Type] name description`. Here, `@param [String] sample_time_field name of the field in `sample` that contains the sample's logical time`

Reply:
Which types should i define for the samples? They are mutable, according to the task output port